src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -461,6 +511,7 @@ protected function getImageExtension($fileName)
         switch ($extension) {
             case 'png':
             case 'gif':
+            case 'jpeg':
```

Comment:
It would also be nice to clean up the way we add the Thumbnail-Urls to the response. If that's done right it would be not that much work to make the subversion per format.